### ISSUE TITLE
Architecture for DANN

### ISSUE THREAD

**Issue by rangwani-harsh** [VisionLearningGroup/DANCE#1]
Dear authors,
Thanks for the code release. I was curious about the fact that accuracy of method DANN (in closed set for office 31) reported in the paper is much higher than accuracies reported for it in other papers like CDAN. So I wanted to know what architecture was used for the classifier and discriminator network along with other hyperparams for your DANN implementation.

Thanks

**Comment by ksaito-ut** [comment]
Hi, thanks for your interest in our work. 

For classifier (C), we used the same architecture as DANCE.
I guess it may improve performance. For the feature extractor, we added one linear layer (nn.Linear(2048, 2048)) on top of ResBase. 
Discriminator was a two-layered model (fc1 = nn.Linear(input_size, 1024), fc2 = nn.Linear(1024, 1)), ReLU was used in fc1.  
The loss was computed as follows. d_s and d_t are the output of discriminator for source and target respectively. 

 dloss_s = torch.mean(d_s ** 2)
 dloss_t = torch.mean((1 - d_t) ** 2)

We did not apply learning rate scheduling for discriminator.

**Comment by rangwani-harsh** [comment]
Thanks for the detailed answer. So the discriminator lr used is 0.01 for all the iterations?

**Comment by ksaito-ut** [comment]
No. Sorry for the confusion. The learning rate is scheduled as is done for feature extractor and classifier.

**Comment by rangwani-harsh** [comment]
Thanks a lot 👍
